components/Badge/badge.css

```diff
@@ -0,0 +1,173 @@
+:root{
+    --log-off:#991b1b;
+}
+
+.badge-container {
+    display: grid;
+    grid-gap: 5rem;
+    padding: 1.5rem 1.5rem;
+    box-shadow: 1px 2px 5px 3px rgba(0, 0, 0, 0.1);
+    grid-template-columns: repeat(4, 2rem);
+}
+
+
+
+.badge {
+    border-radius: 0.2rem;
+    font-size: 0.8em;
+    margin: 0.2rem 0.2rem;
+    padding: 0.5rem 0.5rem;
+    display: flex;
+    justify-content: left;
+    align-items: left;
+    min-height: 0.5rem;
+    width: 8rem; 
+}
+
+
+
+#badge .fa-home, .fa-heart-o, .fa-shopping-cart {
+    position: relative;
+    font-size: 2.5rem;
+}
```

Comment:
Instead of using font-awesome class names you can give a common class name to the icons

Reply:
Sure. I will try that.